common/views/components/BaseTabs/BaseTabs.tsx

```diff
@@ -0,0 +1,140 @@
+import { useState, useRef, useContext, KeyboardEvent } from 'react';
+import { AppContext } from '@weco/common/views/components/AppContext/AppContext';
+import styled from 'styled-components';
+import { classNames } from '@weco/common/utils/classnames';
+
+const TabList = styled.div.attrs({
+  role: 'tablist',
+})``;
+
+type TabProps = {
+  isActive: boolean;
+  tabPanelId: string;
+};
+
+const Tab = styled.button.attrs((props: TabProps) => ({
+  className: classNames({
+    'plain-button no-padding': true,
+  }),
+  role: 'tab',
+  tabIndex: props.isActive ? 0 : -1,
+  'aria-selected': props.isActive,
+  'aria-controls': props.tabPanelId,
+}))<TabProps>`
```

Comment:
Doesn't this type param propagate to [here](https://github.com/wellcomecollection/wellcomecollection.org/pull/5614/files#diff-77dad37477d02a6b483961b2541ef8ff7b0e727fcd9d33671d7c0c6e173763f8R15) - so we don't have to do it twice?

Reply:
it doesn't seem to, no – the one above only deals with what's in the `attrs`

Reply:
![Oct-26-2020 16-32-03](https://user-images.githubusercontent.com/1394592/97200239-ddea5a80-17a8-11eb-93f6-092e98f02c52.gif)